packages/blocks/ternary/src/lib/common/case-types-property.ts

```diff
@@ -0,0 +1,17 @@
+import { Property, StaticDropdownProperty } from '@openops/blocks-framework';
+import caseTypes from '../api-filters/case-types.json';
+
+export function getCaseTypesProperty(): StaticDropdownProperty<string, true> {
+  return Property.StaticDropdown({
+    displayName: 'Case type',
```

Comment:
let's align on property names in all the actions here, I'm not sure if it's Title Case in other places